templates/download/server/provisioning.html

```diff
@@ -167,20 +167,4 @@ <h2>Technical documentation</h2>
 {% endblock content %}
 
 {% block footer_extra %}
```

Comment:
If `{% block footer_extra %}` is empty, it's probably cleaner to simply remove it, no?